### DISCUSSION TITLE
Need help with update file json format

### DISCUSSION THREAD

**Discussion by privatedev11** [tauri-apps/tauri#6006] [Q&A] [answered]
With [update file via json](https://tauri.app/v1/guides/distribution/updater/#update-file-json-format) for updater, do I put that string of code (the one in the link) in my ``tauri.conf.json`` or a separate file? Thanks everyone.

**Comment by FabianLars** [comment, marked as answer]
The json file in that link is what your server/file host should respond with (unless you choose the other response format). It does _not_ belong in tauri.conf.json.

**Reply by privatedev11** [reply]
So if I am using GitHub releases as the 'host' for the binaries, do I just make a .json file with that code which I update every version?

**Reply by FabianLars** [reply]
Yep, and if you use `tauri-action` to create your release and its artifacts you may want to try out its dev branch where a json file generator was implemented

**Reply by privatedev11** [reply]
I don't use tauri-action as I could never get it working. Where do I put the json anyway, the docs aren't very clear.

**Reply by FabianLars** [reply]
The docs can't really help with that because it completely depends on the storage you use. In case of github you can either add it to the release, and use `https://github.com/username/reponame/releases/latest/download/your-updater-file.json` as the endpoint url in tauri.conf.
Alternatively you could host it as a `gist` and point to its "raw" version. Same should work if you just plainly commit it to the repo.

**Reply by privatedev11** [reply]
Thank you!
